sequencer/dbmanager.go

```diff
@@ -181,6 +181,13 @@ func (d *dbManager) storeProcessedTxAndDeleteFromPool() {
 	for {
 		txToStore := <-d.txsStore.Ch
 		d.checkIfReorg()
+
+		// Flush the state db
+		err := d.state.FlushMerkleTree(d.ctx)
+		if err != nil {
+			log.Fatalf("StoreProcessedTxAndDeleteFromPool: %v", err)
```

Comment:
Could be we need to add in this log that the error is when calling the FlushMerkleTree function

Reply:
Sure